okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/UrlSelectorImpl.java

```diff
@@ -155,12 +157,26 @@ private static String switchWsToHttp(String url) {
         return redirectTo(current, baseUrls.get((currentUrl.get() + 1) % baseUrls.size()));
     }
 
+    @Override
+    public Optional<HttpUrl> redirectToHash(HttpUrl current, Optional<String> nodePinValue) {
```

Comment:
We are also missing code changes in #redirectToNext above: RemotingOkHttpCall uses that for retries for requests that fail in #enqueueInternal; as it stands it would just use the currentUrl pinning, which is not what we want.

What we actually want is not clear to me: @ellisjoe what is the retry behavior here, because we have multiple retry paths (at the okhttp layer and application layer in RemotingOkHttpCall).

My intuition tells me we actually don't want to retry on a different host in this setup, instead we should fail the entire chain of requests and make the whole thing retry?